app/client/templates/views/account.js

```diff
@@ -4,11 +4,82 @@ Template Controllers
 @module Templates
 */
 
+/**
+Watches custom events
+
+@param {Object} newDocument  the account object with .jsonInterface
+*/
+var addLogWatching = function(newDocument){
+    var contractInstance = web3.eth.contract(newDocument.jsonInterface).at(newDocument.address);
```

Comment:
I've seen it. So you're saying this is not happening with you? Because it happens to me, but it can be a separate issue..